src/CoamClient.js

```diff
@@ -405,8 +405,9 @@ class CoamClient {
         return this.__exec(data);
     }
 
-    getUserPermissionsForResourceType(principal, resourceType) {
-        let url = this.__buildUrl(`/auth/access-management/v1/principals/{{principal}}/permissions/{{resourceType}}`, {
+    getUserPermissionsForResourceType(principal, resourceType, include = true, permissionFilters = null) {
+        let urlString = `/auth/access-management/v1/principals/{{principal}}/permissions/{{resourceType}}`.concat(`?include=${include}`).concat(permissionFilters ? `&permissionFilter=${permissionFilters}`:'');
```

Comment:
No need for the first `concat` since you're always including the `include` parameter.